lib/read/test/Cardano/Wallet/Read/EraValueSpec.hs

```diff
@@ -28,17 +30,27 @@ import Test.Hspec
     , shouldSatisfy
     )
 import Test.QuickCheck
-    ( Gen
+    ( Arbitrary (..)
+    , Gen
     , Testable (..)
+    , conjoin
+    , counterexample
     , elements
     , forAll
+    , (===)
     )
 
+genEra :: Gen (EraValue Era)
+genEra = do
+    ix <- elements [0 :: Int .. 6]
```

Comment:
This `6` could probably easily be forgotten to be updated, resulting in silently reduced coverage. Can we do something else?

Reply:
I completely agree. I have changed it to `elements knownEras`, which is much clearer anyway.